src/go/rpk/pkg/config/rpk_yaml.go

```diff
@@ -131,28 +134,34 @@ type (
 
 	RpkProfile struct {
 		Name         string               `json:"name" yaml:"name"`
-		Description  string               `json:"description,omitempty" yaml:"description,omitempty"`
-		Prompt       string               `json:"prompt,omitempty" yaml:"prompt,omitempty"`
-		FromCloud    bool                 `json:"from_cloud,omitempty" yaml:"from_cloud,omitempty"`
-		CloudCluster *RpkCloudCluster     `json:"cloud_cluster,omitempty" yaml:"cloud_cluster,omitempty"`
-		KafkaAPI     RpkKafkaAPI          `json:"kafka_api,omitempty" yaml:"kafka_api,omitempty"`
-		AdminAPI     RpkAdminAPI          `json:"admin_api,omitempty" yaml:"admin_api,omitempty"`
-		SR           RpkSchemaRegistryAPI `json:"schema_registry,omitempty" yaml:"schema_registry,omitempty"`
+		Description  string               `json:"description" yaml:"description"`
+		Prompt       string               `json:"prompt" yaml:"prompt"`
+		FromCloud    bool                 `json:"from_cloud" yaml:"from_cloud"`
+		CloudCluster RpkCloudCluster      `json:"cloud_cluster" yaml:"cloud_cluster"`
+		KafkaAPI     RpkKafkaAPI          `json:"kafka_api" yaml:"kafka_api"`
+		AdminAPI     RpkAdminAPI          `json:"admin_api" yaml:"admin_api"`
+		SR           RpkSchemaRegistryAPI `json:"schema_registry" yaml:"schema_registry"`
 
 		// We stash the config struct itself so that we can provide
 		// the logger / dev overrides.
 		c *Config
 	}
 
 	RpkCloudCluster struct {
-		Namespace string `json:"namespace" yaml:"namespace"`
-		Cluster   string `json:"cluster" yaml:"cluster"`
-		Auth      string `json:"auth" yaml:"auth"`
+		Namespace   string `json:"namespace" yaml:"namespace"`
+		ClusterID   string `json:"cluster_id" yaml:"cluster_id"`
+		ClusterName string `json:"cluster_name" yaml:"cluster_name"`
+		AuthOrgID   string `json:"auth_org_id" yaml:"auth_org_id"`
+		AuthKind    string `json:"auth_kind" yaml:"auth_kind"`
```

Comment:
Can we add `,omitempty` it's weird seeing a cloud_cluster field in a self hosted cluster:

```
$ rpk profile create default
$ rpk profile print
name: default
description: ""
prompt: ""
from_cloud: false
cloud_cluster:
    namespace: ""
    cluster_id: ""
    cluster_name: ""
    auth_org_id: ""
    auth_kind: ""

```